rider/protocol/src/kotlin/model/frontendBackend/GodotFrontendBackendModel.kt

```diff
@@ -34,13 +45,18 @@ object GodotFrontendBackendModel : Ext(SolutionModel.Solution) {
 
         // Misc backend/fronted context
         property("godotPath", string).documentation = "Path to GodotEditor"
-        property("mainProjectBasePath", string).documentation = "Path to the folder with the project.godot"
 
         // Settings stored in the backend
         field("backendSettings", aggregatedef("GodotBackendSettings") {
             property("enableDebuggerExtensions", bool)
+
+            property("lspConnectionMode", LanguageServerConnectionMode)
+            property("remoteHostPort", int)
+            property("useDynamicPort", bool)
         })
 
+        property("godotDescriptor", GodotDescriptor)
```

Comment:
Request ProjectModel team to read the Godot SDK version.

Reply:
https://youtrack.jetbrains.com/issue/RIDER-104684/Please-provide-Project-Sdk-name-and-version-in-the-project.ProjectProperties